app/javascript/gobierto_visualizations/modules/subsidies_controller.js

```diff
@@ -135,23 +141,17 @@ export class SubsidiesController {
           ? parseFloat(subsidy.amount)
           : 0.0;
 
-      let [beneficiary_id, ...beneficiary_name] = subsidy.beneficiary.split(
-        " "
-      );
+      let [beneficiary_id, ...beneficiary_name] = subsidy.beneficiary.split(" ");
```

Comment:
Are you sure `beneficiary` always come as string?

Actually this whole `for` block, deserves an important refactorization, replacing the `for` loop with a `map`, and providing fallbacks for every single prop. For instance:

```js
subsidiesData.map(({ amount = 0, beneficiary = "", ...rest }) => {
  // any complex logic you need before returning

  return {
    ...rest,
    amount: ...,
    range: ...,
    // etc...
  }
})
```

Reply:
Done!